packages/bpk-component-breakpoint/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "bpk-mixins": "^17.0.7",
     "prop-types": "^15.5.8",
-    "react-responsive": "^1.3.2"
+    "react-responsive": "^3.0.0"
```

Comment:
This looks like a significant change. Two new major versions, did we review the CHANELOG.md for react-responsive and determine there are no breaking changes we need to address?

Reply:
Had a look at CHANGELOG.md, don't think there is any breaking change. Also ran storybook locally, everything seems fine.